### PR TITLE
Panel refactor rows

### PR DESCRIPTION
**Pull request by msecret**
Refactors the rows by creating new components to replace them. I currently haven't refactored the complex usage and limits UI, that might need another component.

### CODE REVIEW

File: src/css/components/elastic_line.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+@import '../override_vars.scss';
+@import '../core.scss';
+@import '../../../node_modules/uswds/src/stylesheets/core/defaults.scss';
+
+.elastic_line {
```
Comment (adborden) [inline src/css/components/elastic_line.scss:6]: good name :)

File: src/css/components/panel-dep.scss [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    margin-bottom: 0;
+    padding-bottom: $panel-padding;
+  }
+
+  .panel-row:last-child {
+    padding-bottom: 0;
+  }
+}
+
+.panel-row-bordered {
```
Comment (adborden) [inline src/css/components/panel-dep.scss]: Can we add an example of this and `-boxed` to the pattern library examples?

**Review by adborden — COMMENTED** [review]
:+1:

File: src/css/components/panel-dep.scss (hunk 2 above)
Comment (msecret) [reply src/css/components/panel-dep.scss]: I'm going to get rid of this file, but will have examples of all new stuff

File: src/css/components/expandable_box.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
+
+@import '../../../node_modules/uswds/src/stylesheets/core/defaults.scss';
+@import '../override_vars.scss';
+@import '../core.scss';
+
+$box-padding: $grid-3;
+
+.expandable_box {
+  background-color: white;
```
Comment (adborden) [inline src/css/components/expandable_box.scss:9]: Is this necessary? We can't just use transparent?

**Review by adborden — APPROVED** [review]

**Pull request merged**